crates/ruff_python_trivia/src/pragmas.rs

```diff
@@ -0,0 +1,30 @@
+/// Returns `true` if a comment appears to be a pragma comment.
+///
+/// ```
+/// assert!(ruff_python_trivia::is_pragma("# type: ignore"));
+/// assert!(ruff_python_trivia::is_pragma("# noqa: F401"));
+/// assert!(ruff_python_trivia::is_pragma("# noqa"));
+/// assert!(ruff_python_trivia::is_pragma("# NoQA"));
+/// assert!(ruff_python_trivia::is_pragma("# nosec"));
+/// assert!(ruff_python_trivia::is_pragma("# nosec B602, B607"));
+/// assert!(ruff_python_trivia::is_pragma("# isort: off"));
+/// assert!(ruff_python_trivia::is_pragma("# isort: skip"));
+/// ```
+pub fn is_pragma(comment: &str) -> bool {
```

Comment:
Nit

```suggestion
pub fn is_pragma_comment(comment: &str) -> bool {
```

To make it clear that it tests the whole comment and isn't just testing whether `text` is a pragma keyword